app/config.py

```
@@ -107,6 +107,10 @@ def database_url(self) -> str:
     # Override via REMOTE_TRANSFER_MAX_CONCURRENT environment variable
     remote_transfer_max_concurrent: int = 2
 
+    # P2P private-network defaults
+    p2p_listen_host: str = "0.0.0.0"
```

Ruff (0.15.11)

[error] Possible binding to all interfaces

(S104)